nats-jetstream-http-connector/README.md

```diff
@@ -12,16 +12,12 @@ The job of the connector is to read messages from the subject in the given strea
 - `CONTENT_TYPE`: Content type used while creating post request
 - `NATS_SERVER`: NATS server address. It can be a remote address `nats://127.0.0.1:4222` or in case deployed in Kubernetes, can reached using corresponding service name
 - `STREAM`: stream from which connector will read messages.
-- `FISSION_CONSUMER`: this is the consumer which fission uses to pull data from stream, proccesses it and pushes it forward to `RESPONSE_TOPIC`
+- `CONSUMER`: this is the consumer which fission uses for monitoring and creating resources(eg, creating pods)
 
 
 
 ## Resources
 * To setup and run nats streaming server, reference https://docs.nats.io/nats-server/installation#installing-on-kubernetes-with-nats-operator
 * For running the connecter with fission e.g.  
 
-``` fission mqt create --name jetstreamtest1 --function helloworld --mqtype nats-jetstream --mqtkind keda --topic input.created --resptopic output.response-topic --errortopic output.error-topic --maxretries 3 --metadata stream=input --metadata fissionConsumer= fission_consumer --metadata natsServerMonitoringEndpoint=nats-jetstream.default.svc.cluster.local:8222  --metadata natsServer=nats://nats-jetstream.default.svc.cluster.local:4222 ```
-
- 
-          
-
+```fission mqt create --name jetstreamtest --function helloworld --mqtype nats-jetstream --mqtkind keda --topic input.created --resptopic output.response-topic --errortopic erroutput.error-topic --maxretries 3 --metadata stream=input --metadata fissionConsumer=fission_consumer --metadata natsServerMonitoringEndpoint=nats-jetstream.default.svc.cluster.local:8222 --metadata natsServer=nats://nats-jetstream.default.svc.cluster.local:4222 --metadata consumer=fission_consumer```
```

Comment:
We are passing `fissionConsumer` and `consumer` as metadata with same value here, should we get rid one of these.